sample-package-core/src/main/resources/OSGI-INF/custombulkaction-bulk-action-contrib.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.sample.core.actions.custombulkaction">
+
+  <require>org.nuxeo.ecm.core.bulk</require>
+
+  <extension target="org.nuxeo.ecm.core.bulk" point="actions">
+    <action name="custombulkaction" bucketSize="100" batchSize="25" sequentialCommands="true"/>
+  </extension>
+
+  <extension target="org.nuxeo.runtime.stream.service" point="streamProcessor">
+    <streamProcessor name="custombulkaction" class="org.nuxeo.sample.core.CustomBulkAction"
+                     defaultConcurrency="1" defaultPartitions="1" logConfig="bulk">
```

Comment:
defaultConcurrency is the number of computation threads per Nuxeo node,
defaultPartitions fixes the maximum concurrency for the computation threads at the Nuxeo Cluster level

Reply:
logConfig removed and defaultConcurrency & defaultPartitions changed following our discussion on them